beacon-chain/blockchain/van_minimal_consensus_info.go

```diff
@@ -0,0 +1,197 @@
+package blockchain
```

Comment:
In my opinion we should move back to the `rpc/beacon` because `helpers` are more like `committees.go` logic than any of helpers. Maybe other solution is to have `core/consensus` like `core/validators` is. What do you think @atif-konasl ?

Reply:
Done. As we have agreed on I've moved to the rpc/beacon package.